### PR TITLE
Added webpack 2 instructions

### PR DESCRIPTION
**Pull request by kazazor**
Added instruction for: https://github.com/rpominov/svgo-loader/issues/6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,33 @@ $ npm install svgo-loader --save-dev
...
+          {convertColors: {shorthex: false}},
+          {convertPathData: false}
+        ]
+      }
+    }
+  ]
+}
+```
+
+Or
```
Comment (rpominov) [inline README.md]: I'd remove this "Or"

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
+.idea
```
Comment (rpominov) [inline .gitignore]: Please remove this change. 

Btw, I usually add things like that to my `~/.gitignore` and they're getting ignored in all git directories.

**Review by rpominov — CHANGES_REQUESTED** [review]
Looks good overall but there're couple comments.

File: .gitignore (hunk 2 above)
Comment (kazazor) [reply .gitignore]: I don't mind doing that here, but in general I think people are working with different IDEs and this is helping them as well..

But I really don't mind :)

**Comment by kazazor** [comment]
@rpominov - made the requested changes and squashed my commits so you'll have less useless commits on master

**Comment by rpominov** [comment]
Great! Thank you!

**Pull request merged**